crates/sui-proxy/src/admin.rs

```diff
@@ -112,16 +113,29 @@ pub fn app(
             .layer(Extension(Arc::new(allower)));
     }
     router
+        // Enforce on all routes.
+        // If the request does not complete within the specified timeout it will be aborted
+        // and a 408 Request Timeout response will be sent.
+        .layer(TimeoutLayer::new(Duration::from_secs(var!(
```

Comment:
Any idea how long these reqs take today?

Reply:
yes, i do.  It around 3 seconds give or take.

https://metrics.sui.io/goto/kvX5MFRHR?orgId=1